scripts/serverless

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+CURRENT_DIR=`dirname $BASH_SOURCE`
+
+# Source env variables
+echo "Sourcing environment"
+if [ -x .env ]; then
+  . ./.env
+  echo "... done"
+else
+  echo "... no .env file found"
+fi
+
+if [ "$DATABASE_URL" = "" ]; then
+  echo "No DATABASE_URL envvar found; cannot continue."
+  exit 1
+fi
+
+rm -rf .serverless
+
+echo "Phase 1: Create serverless package"
+sls package
+
+echo "Phase 2: Build webpack bundle and add it to serverless package"
+scripts/build
+scripts/generate-cache
+scripts/bundle
+mv $CURRENT_DIR/../lambda.zip $CURRENT_DIR/../.serverless/graphql.zip
```

Comment:
This refers to `$CURRENT_DIR/../.serverless` whereas on line 18 we just rm `.serverless`; these should probably be consistent?

Reply:
Yes.